### ISSUE TITLE
Swagger UI not accessible

### ISSUE THREAD

**Issue by marcauberer** [livepoll/live-poll-api#18]
At the moment, the Swagger UI page '/swagger-ui/index.html' and the redirect from '/' are not accessible anymore due to the current security configuration.
Would love to see this fixed ...

**Comment by Niklas-23** [comment]
I tried to solve the problem and found that we have two problems with Swagger. The first is that I removed authentication from the Swagger UI, but it only works in Postman. I don't know why, but when I try to open the same link in the browser "permitAll()" is ignored. In Postman i receive the html file. The second problem is that the redirect for "/" doesn't work. The Swagger UI can only be accessed through "/swagger-ui/index.html". The redirection triggers the following error: 

"Could not resolve view with name 'redirect:/swagger-ui/index.html' in servlet with name 'dispatcherServlet'"
